tests/fdb/conftest.py

```diff
@@ -0,0 +1,15 @@
+from args.populate_fdb_args import add_populate_fdb_args
+from common.fixtures.populate_fdb import populate_fdb
+
+# FDB pytest arguments
+def pytest_addoption(parser):
+    """
+        Adds option to FDB pytest
+
+        Args:
+            parser: pytest parser object
+
+        Returns:
+            None
+    """
+    add_populate_fdb_args(parser)
```

Comment:
So when we originally talked about putting arguments in a separate file I don't think I realized that each feature could have their own `conftest` with their own `addoption` method. 😅 I think we could probably get away with having the arguments defined directly in the `addoption` method. What do you think?

Reply:
I think having the args groups in separate file is a good housekeeping. Consider the case where we have multiple test cases for one feature, the args file can grow big and also args could get mingled to a point deleting might require verifying no other test case is affect.

Reply:
That's a fair point, let's leave it as is!